### DISCUSSION TITLE
Github 권한 부여 부탁드립니다

### DISCUSSION THREAD

**Discussion by jasonjo97** [Pseudo-Lab/9th-builder#35] [Issue and Q&A]
찬란님 안녕하세요!
혹시 이번 기수에 새로 조인하신 해당 맴버들 깃헙 계정 권한 부여해줄 수 있을까요?? 
디스코드 계정도 같이 전달드립니다. 감사합니다!

- 류지혜 Domwis-IR (yihaeroo)
- 유경민 yukyeongmin (pixeldosa)
- 신찬영 ChanYeongShin (eren_ai)
- 이중원 oneoftwo (oneoftwo942)

**Comment by seriousran** [comment]
초대 완료하였습니다! 1주일 내로 수락해주셔야합니다!

**Reply by jasonjo97** [reply]
찬란님 연말 잘 보내고 계신지요오 
경민님이 이전에 초대 수락을 못했었더라고요 편하실때 한번만 다시 초대 부탁드리겠습니다 감사합니다!

- 유경민 yukyeongmin (pixeldosa)
